scripts/browserstack/start_tunnel.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
Could you add comment up top here explaining what the purpose of this script is?